go.mod

```diff
@@ -133,7 +133,7 @@ require (
 )
 
 replace (
-	github.com/docker/cli => github.com/docker/cli v20.10.3-0.20220309205733-2b52f62e9627+incompatible
+	github.com/docker/cli => github.com/ndeloof/cli v0.0.0-20220317134108-66942b8c2379
```

Comment:
Hmmm... This if I understood well is related to `manager.Metadata#Completion`. The thing is that this is not merged in `docker/cli` yet... 

I'm also not sure if the `SchemaVersion` should change too.

Reply:
Also... If we are already changing the struct, that would be great if we could add a `Meta: map[string]string` to open for other applications.

Reply:
this one is actually obsolete, in docker/cli we gave up with a new schema, and just run the `__complete` command on plugins. If they don't implement this command, it will just fail and offer no completion.
Also, the idea of `SchemaVersion` is to avoid an open `map[string]string` that would make it unpredictable about the possible values being set by plugin.